lib/scala/syntax/specialization/js/src/main/scala/org/enso/syntax/text/Main.scala

```diff
@@ -31,4 +35,19 @@ object Parse {
       case e: Throwable => throw js.JavaScriptException(e.getMessage)
     }
   }
+  @JSExportTopLevel("doc_parser_generate_html_source")
+  def doc_parser_generate_html_source(program: String): String = {
+    try {
+      val parser   = new Parser()
+      val module   = parser.run(program)
+      val dropMeta = parser.dropMacroMeta(module)
+      val doc      = DocParserRunner.createDocs(dropMeta)
+      val htmlCode = DocParserHTMLGenerator.generateHTMLForEveryDocumented(doc)
+      htmlCode
+    } catch {
+      // FIXME We wrap the error message in JavaScriptException, so that javascript
```

Comment:
We're on ScalaJS 1.x, so please test if this is still needed.

Reply:
Tested with IDE, seems to work just fine

Reply:
And tested in browser, produces HTML code from documented code, just as expected.